### ISSUE TITLE
P2: Issues with editing more than on post on front-end, Gutenberg styles

### ISSUE THREAD

**Issue by sedocirp7** [Automattic/themes#523]
# Issue 1 - Data wipeout on P2 post's edit mode

## Steps to replicate
- Go to any P2 and click on the three dots on a specific P2 Post to edit it.
 
- Now keep the Editor open in the **First P2 Post** and click the edit icon from the three dots on a **Second P2 Post**. I usually do this to copy paste format or any recurring data. 

## Result
- The editor on the **First P2** vanishes as soon as the **Second P2's** editor is open and there is nothing saved in the draft section too on the **First P2**. 

- The data gets wiped out for the **First P2** and there is nothing we can do to get it back as this is edited on the P2 editor and not the Calypso editor. 

## Expected
There should be a provision for both the P2 post's editors to be opened simultaneously or at least a prompt which says 

> Only one P2 post can be edited at a time. 

I have lost 2 hours of work on this simple bug. 

Here is a gif on the issue - https://gifyu.com/image/wYfa

## Other Testing

The provision to edit multiple P2 Posts/comments is there in the **P2 Classic Theme**. I would like to see this feature in the latest P2 Theme too. 

Here is the gif on the P2 Classic Theme - https://gifyu.com/image/wYfX

I found this bug quite annoying as there is nothing mentioned about it on the P2 theme's info page as well. 

# Issue 2 - Photo alignment doesn't work in P2 Post for Photos with a caption

## Steps to replicate
- Go to the Calypso editor with the Gutenberg editor *(Yes, I switched to the Calypso editor instead of editing posts directly from the P2 editor following the bug)* and add some content followed by few Photos. 
-  Put a caption for some photos and make every photo's alignment as "Center" . 
- Publish the page and view the page. 

## Result
The captioned photos lose its alignment in the site/blog page. 

## Expected
The captioned photos should retain its alignment irrespective of the caption being there or not being there. The captioned photos are aligned fine on a non-P2 theme. 

Here's a comparison of the same data on the P2 theme and a non P2 theme - https://gifyu.com/image/wYtY

**Comment by laurelfulford** [comment]
> Data wipeout on P2 post's edit mode

I was able to recreate this issue.

> Photo alignment doesn't work in P2 Post for Photos with a caption

I haven't been able to recreate this one -- I tried both larger and smaller images, centred, with captions, but both seem to display as expected:

![image](https://user-images.githubusercontent.com/177561/52239946-7ed20d00-2884-11e9-80ec-5f1a951c9bf6.png)

We did have a problem with other theme's CSS overriding the image block styles, if they were set to `display: block` -- the image block is `display: table`. It's possible that something changed in Gutenberg since this was reported -- P2 does set `.aligncenter` to be `display: block`, but the more specific `.wp-block-image .aligncenter {display: table;}` coming from Gutenberg is overriding it.
